### PR TITLE
NOTRACK - Redirect to youclap.tech... 👋

### PR DESCRIPTION
**Pull request by portellaa**

### CODE REVIEW

**Review by tallnato — APPROVED** [review]
LGTM 🚀 🤘

**Review by hugomsilvam — APPROVED** [review]
goooooo

File: contact.html [hunk 1]
```diff
@@ -1,137 +1,7 @@
...
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.17.0/jquery.validate.min.js"></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" async></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/animejs/1.0.0/anime.min.js" async></script>
-        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-one-page-nav/3.0.0/jquery.nav.min.js"></script>
-        <script src="js/contact.min.js" async></script>
-        <script src="js/script.min.js" async></script>
-    </body>
+    <p>Redirecting to <b>youclap.tech</b>... ⏳</p>
```
Comment (ruimartinsptl) [inline contact.html:6]: I think that you should put "youclap.tech" as a link.
Only for case of an error occurs, after the page load and before the redirect.

Comment (portellaa) [reply contact.html:6]: i will open a new PR with that.

**Pull request merged**